crypto/x86_64cpuid.pl

```diff
@@ -434,21 +434,6 @@
 sub gen_random {
 my $rdop = shift;
 print<<___;
-.globl	OPENSSL_ia32_${rdop}
-.type	OPENSSL_ia32_${rdop},\@abi-omnipotent
-.align	16
-OPENSSL_ia32_${rdop}:
-	mov	\$8,%ecx
-.Loop_${rdop}:
-	${rdop}	%rax
-	jc	.Lbreak_${rdop}
-	loop	.Loop_${rdop}
-.Lbreak_${rdop}:
-	cmp	\$0,%rax
-	cmove	%rcx,%rax
-	ret
-.size	OPENSSL_ia32_${rdop},.-OPENSSL_ia32_${rdop}
```

Comment:
@dot-asm: Just to be sure, are you happy with just keeping *_bytes versions? As far as I can see, it was never in a public header and only used by eng_rdrand.

Reply:
The things is that _bytes versions were added *specifically* to resolve the shortcoming[s] of the original versions. If anything is to go [at any point], then it should be original ones. As for making them public. Well, one can do that, but does one have to? eng_rdrand is internal engine and is not limited by symbols exported by libcrypto.so.

Reply:
I think I've tried to answer wrong question. It's not about making something public, but about removal of symbol that wasn't public and now not used. Sure, why should removal be a problem.

Reply:
Theoretically, someone linking openssl as a static library could be depending on this symbol. But this is unlikely since it'll break on shared library builds on Linux and (I think) Windows anyway.